src/import/engine.rs

```diff
@@ -34,7 +34,7 @@ use storage::mvcc::{Write, WriteType};
 use storage::types::{split_encoded_key_on_ts, Key};
 use storage::{is_short_value, CF_DEFAULT, CF_WRITE};
 use util::config::MB;
-use util::rocksdb::properties::{SizeProperties, SizePropertiesCollectorFactory};
+use util::rocksdb::properties::{RangeProperties, RangePropertiesCollectorFactory};
```

Comment:
I think you can leave the `SizeProperties` under `src/import` for now.
@DorianZheng You can use `RangeProperties` to split regions by the number of keys in tikv-importer when necessary.